test/functional/p2p_invalid_messages.py

```diff
@@ -24,8 +27,8 @@
     wait_until,
 )
 
-MSG_LIMIT = 4 * 1000 * 1000  # 4MB, per MAX_PROTOCOL_MESSAGE_LENGTH
-VALID_DATA_LIMIT = MSG_LIMIT - 5  # Account for the 5-byte length prefix
+VALID_DATA_LIMIT = MAX_PROTOCOL_MESSAGE_LENGTH - 5  # Account for the 5-byte length prefix
+DISCONNECTING = 'disconnecting peer='
```

Comment:
My preference is not to test the debug log for actions that we can test directly. Here, we can directly test that the peer has been disconnected, so I don't think we need to test for this message in the debug log.

Reply:
I added this after noticing both here and in the #19107 that some of the tests aren't testing for the disconnection, and it's a free additional check. If a change made one of these disconnections no longer be logged without warning and no test needing to be changed, ISTM that could be a regression.

Reply:
Right - I think the fix is to test for disconnection, not to test for the log message that accompanies disconnection.

Reply:
Ok, in three cases it was only logging the disconnection because `disconnect_p2ps()` was previously being called in the scope of the `assert_debug_log`. Updating along with my review feedback in #19107.

Reply:
Done -- good call, thanks.